apollo-router/src/cache/redis.rs

```diff
@@ -2,6 +2,7 @@
 
 use std::fmt;
 use std::sync::Arc;
+use std::time::Duration;
```

Comment:
Strange. I can't see why this change is here and `clippy` isn't complaining about it. Same for the timeout use...

Reply:
because the entire module is under the `experimental_cache` cargo feature. Fixed in 48d4e29b